gen/calc.py

```diff
@@ -414,7 +414,7 @@ def calculate_config_yaml(user_arguments):
 
 
 def calculate_mesos_isolation(enable_gpu_isolation):
-    isolators = ('cgroups/cpu,cgroups/mem,disk/du,network/cni,filesystem/linux,'
+    isolators = ('cgroups/cpu,cgroups/mem,cgroups/blkio,disk/du,network/cni,filesystem/linux,'
```

Comment:
@adam-mesos it should be throttling, not killing. However I am also not sure about defaults and hence if we are already throttling by default without writing the the CGroup nodes. Hence I totally agree that we need to make this more transparent to the team and the users before enabling it within DC/OS.